src/coreclr/vm/ceeload.cpp

```diff
@@ -1552,10 +1552,10 @@ DWORD Module::AllocateDynamicEntry(MethodTable *pMT)
 
     DWORD newId = InterlockedExchangeAdd((LONG*)&m_cDynamicEntries, 1);
```

Comment:
This can be moved under the lock and changed to regular increment.

Reply:
Fixed. Thank you.